src/ChatHeader.css

```diff
@@ -0,0 +1,48 @@
+.chatHeader__container{
+    display: flex;
+    justify-content: space-between;
+    flex-direction: row;
+    align-items: center;
+    background-color: lightgrey;
+    border-left:1px solid darkgray;
+    border-bottom:1px solid darkgray;
+    flex:1.24;
```

Comment:
no need of 
justify-content
flex-direction
background-color
border-left
flex

add padding

Reply:
done